kod/object/item/passitem/ring/ringinv.kod

```diff
@@ -62,7 +63,12 @@ messages:
 	      return False;
       }
 
-      propagate;
+      if send(poOwner,@GetKarma) > -50  
+      { 
+         send(poOwner,@MsgSendUser,#message_rsc=ringinvisibility_cant);
```

Comment:
@skittles1 Could you elaborate on exactly how to use the ReqUse() and propogate please, code example would be nice. Thank you kindly